package.json

```diff
@@ -29,8 +29,20 @@
         "properties": {
           "extension-test-runner.wrapper": {
             "markdownDescription": "A wrapper script that runs tests. When running tests outside of debug mode, arguments will be passed to this script, and it should spawn the first argument as a child process.\n\nA common use case for this is to set it to `xvfb-run` for use in remotes which lack a display.",
-            "examples": ["xvfb-run"],
-            "type": "string"
+            "examples": [
+              "xvfb-run"
+            ],
+            "oneOf": [
+              {
+                "type": "string"
```

Comment:
This type of setting wouldn't render in the Settings editor. Would it make sense to migrate over string values to a single-element array or to keep the setting as a string but split the value more carefully?